tests/contrib/operators/test_singularity_operator.py

```diff
@@ -17,14 +17,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import unittest
-
 from parameterized import parameterized
-import six
 from spython.instance import Instance
-
 from airflow import AirflowException
 
+import unittest
```

Comment:
Old order import is better. 
Reference: https://github.com/apache/airflow/pull/4892